sql/parse/parse.go

```diff
@@ -400,7 +400,7 @@ func convertCreateTable(c *sqlparser.DDL) (sql.Node, error) {
 		sql.UnresolvedDatabase(""), c.Table.Name.String(), schema, c.IfNotExists), nil
 }
 
-func convertCreateView(ctx *sql.Context, c *sqlparser.DDL) (sql.Node, error) {
+func convertCreateView(ctx *sql.Context, query string, c *sqlparser.DDL) (sql.Node, error) {
```

Comment:
You should have some test cases of this in parse_test.go. Looks like the only test of CREATE VIEW right now is for an error case?